Makefile

```diff
@@ -45,7 +50,10 @@ ui: swagger_client
 	src/apps/keyvis/download_dummydata.sh &&\
 	REACT_APP_DASHBOARD_API_URL="" npm run build
 
-server:
+template:
+	scripts/generate_diagnose_report_template.sh
```

Comment:
Prefer to call it `embed_template` for two reasons:
- It embeds template into the source code instead of generates a new template
- We may have other templates in future.